.github/workflows/build_centos7.yml

```diff
@@ -95,7 +96,11 @@ jobs:
         if: github.event_name == 'release' && github.event.action == 'created'
         id: get_release
         uses: bruceadams/get-release@v1.3.2
-        
+
+      - name: Config OR-Tools URL
+        run: |
+          echo "ORTOOLS_URL=https://github.com/rte-france/or-tools/releases/download/v9.8-rte1.0/ortools_cxx_centos7_static_sirius.zip" >> $GITHUB_ENV
```

Comment:
to be updated check that --> [message](https://teams.microsoft.com/l/message/19:554ba80917c8496785fdf5f38cb127dd@thread.v2/1721834224132?context=%7B%22contextType%22%3A%22chat%22%7D)